sdk/src/main/java/io/opentelemetry/sdk/trace/SpanBuilderSdk.java

```diff
@@ -238,7 +238,6 @@ public Span startSpan() {
   private static Clock getClock(Span parent, Clock clock) {
     if (parent instanceof RecordEventsReadableSpan) {
       RecordEventsReadableSpan parentRecordEventsSpan = (RecordEventsReadableSpan) parent;
-      parentRecordEventsSpan.addChild();
```

Comment:
Nice :) we had the logic of num children in a `getClock` method :))

Reply:
yeah, I had to scratch my head at this one a bit. :)